lib/events.js

```diff
@@ -31,6 +32,9 @@ module.exports = EventEmitter;
 // Backwards-compat with node 0.10.x
 EventEmitter.EventEmitter = EventEmitter;
 
+EventEmitter.usingDomains = false;
```

Comment:
To be clear, I’m basically suggesting only taking this line of the diff, and leaving the rest alone

Reply:
I'm open to that as well, wasn't sure the context or why that line was removed. Should we add that line back to master instead?

Would doing another release next tuesday make sense then or should we get this out asap

Reply:
👍 on this

Reply:
@MylesBorins I guess it seems fair to remove the line because it would be false-y by default anyway, so explicitly setting it here and thereby leaking domains logic into this module seems undesirable.

> Would doing another release next tuesday make sense then or should we get this out asap

It’s not really all that clear to me just *how* breaking this change is or how many people it would affect – but if we’re only adding this line back, I don’t see a reason not to do a release as soon as we can.

Reply:
Ok opening a PR